enterprise/server/auditlog/auditlog.go

```diff
@@ -243,14 +243,14 @@ func (l *Logger) GetLogs(ctx context.Context, req *alpb.GetAuditLogsRequest) (*a
 		return nil, err
 	}
 
-	if err := authutil.AuthorizeOrgAdmin(u, u.GetGroupID()); err != nil {
+	if err := authutil.AuthorizeOrgAdmin(u, req.GetRequestContext().GetGroupId()); err != nil {
```

Comment:
hm I'm a bit confused why this is needed - I think `u.GetGroupID()` should in theory be returning the user's selected group ID based on the request context?

Reply:
Good point. It's probably an oversight but the request context is not respected for calls using an API key. Fixing that is prooooobably the better way to go.

Reply:
ah yeah, I think at the time, I wasn't thinking that people would call the HTTP endpoints using an API key.